spec/index.bs

```diff
@@ -6736,10 +6736,20 @@ Issue: Because timestamp query provides high-resolution GPU timestamp, we need t
 interface GPUCanvasContext {
     GPUSwapChain configureSwapChain(GPUSwapChainDescriptor descriptor);
 
-    Promise<GPUTextureFormat> getSwapChainPreferredFormat(GPUDevice device);
+    GPUTextureFormat getSwapChainPreferredFormat(GPUAdapter adapter);
 };
 </script>
 
+{{GPUCanvasContext}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for=GPUCanvasContext>
+    : <dfn>\[[preferred_format]]</dfn>, of type [=ordered map=]&lt;{{GPUAdapter}}, {{GPUTextureFormat}}&gt;
```

Comment:
Do we need to specify this internal slot at all? Can we say the magic of producing the result value is implementation-dependent?

Reply:
I considered it, and this value made more sense when the preferred value was appended to the supported list, not already included in it. The primary reason for keeping it was to ensure algorithmically that the preferred values is always the same for any given canvas/adapter pairing.

Reply:
See https://github.com/gpuweb/gpuweb/pull/1185#discussion_r520926064